docs/source/configuration.rst

```diff
@@ -103,6 +103,12 @@ pose_publish_period_sec
   Interval in seconds at which to publish poses, e.g. 5e-3 for a frequency of
   200 Hz.
 
+publish_to_tf
+  Enable or disable providing of TF transforms.
+
+publish_tracked_pose_msg
+  Enable publishing of tracked pose as a `geometry_msgs/PoseStamped` to topic "tracked_pose".
```

Comment:
Double backticks are needed in RST, as you did in the other file.

Reply:
The other messages here in this file are links, so we should probably also add a link for PoseStamped.

Reply:
I opted for
```
``geometry_msgs/PoseStamped`_
```
to be consistent with the rest of this file. But forgot the underscore before, sorry!

Reply:
oh I only just realized what this link does. Will add that too!

Reply:
yes, RST is quite unituitive with simple inline links :smile: If in doubt, you can also build the html version locally using `sphinx-build docs/ out/` and check it.

Thank you